### PR TITLE
[BPK-3168] Add masked phone input example

### PR DESCRIPTION
**Pull request by georgegillams**
+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] [Links platform metadata](https://github.com/Skyscanner/backpack-docs/blob/master/docs/src/layouts/links.js)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: docs/src/pages/PhoneInputPage/PhoneInputPage.js [hunk 1]
```diff
@@ -80,10 +80,10 @@ class StoryContainer extends Component<Props, State> {
         value={this.state.value}
         dialingCode={this.state.dialingCode}
         dialingCodes={[
-          { code: '1', description: '+1 (US)' },
-          { code: '2', description: '+1 (CA)' },
-          { code: '44', description: '+44' },
-          { code: '55', description: '+55' },
+          { code: '1', description: '+1 (US)', numberPrefix: '+1' },
+          { code: '1_ca', description: '+1 (CA)', numberPrefix: '+1' },
+          { code: '44', description: '+44', numberPrefix: '+1' },
```
Comment (ojcurtis) [inline docs/src/pages/PhoneInputPage/PhoneInputPage.js]: Should this be `+44`

File: docs/src/pages/PhoneInputPage/PhoneInputPage.js [hunk 2]
```diff
@@ -80,10 +80,10 @@ class StoryContainer extends Component<Props, State> {
...
+          { code: '55', description: '+55', numberPrefix: '+1' },
```
Comment (ojcurtis) [inline docs/src/pages/PhoneInputPage/PhoneInputPage.js]: Should this be `+55`

File: docs/src/pages/PhoneInputPage/PhoneInputPage.js [hunk 3]
```diff
@@ -80,10 +80,10 @@ class StoryContainer extends Component<Props, State> {
         value={this.state.value}
         dialingCode={this.state.dialingCode}
         dialingCodes={[
-          { code: '1', description: '+1 (US)' },
-          { code: '2', description: '+1 (CA)' },
-          { code: '44', description: '+44' },
-          { code: '55', description: '+55' },
+          { code: '1', description: '+1 (US)', numberPrefix: '+1' },
```
Comment (ojcurtis) [inline docs/src/pages/PhoneInputPage/PhoneInputPage.js]: Do the codes need to be aligned? E.g. `1_us`, `1_ca`, `44_uk`?

File: docs/src/pages/PhoneInputPage/PhoneInputPage.js (hunk 1 above)
Comment (georgegillams) [reply docs/src/pages/PhoneInputPage/PhoneInputPage.js]: 🤦‍♂️

File: docs/src/pages/PhoneInputPage/PhoneInputPage.js (hunk 3 above)
Comment (georgegillams) [reply docs/src/pages/PhoneInputPage/PhoneInputPage.js]: As long as each is unique it doesn't matter, but can't do any harm 👍

Comment (ojcurtis) [reply docs/src/pages/PhoneInputPage/PhoneInputPage.js]: 👍

File: docs/src/pages/PhoneInputPage/PhoneInputPage.js [hunk 4]
```diff
@@ -80,10 +80,10 @@ class StoryContainer extends Component<Props, State> {
         value={this.state.value}
         dialingCode={this.state.dialingCode}
         dialingCodes={[
-          { code: '1', description: '+1 (US)' },
-          { code: '2', description: '+1 (CA)' },
-          { code: '44', description: '+44' },
-          { code: '55', description: '+55' },
+          { code: '1_us', description: '+1 (US)', numberPrefix: '+1' },
+          { code: '1_ca', description: '+1 (CA)', numberPrefix: '+1' },
+          { code: '44_br', description: '+44 (BR)', numberPrefix: '+44' },
```
Comment (ojcurtis) [inline docs/src/pages/PhoneInputPage/PhoneInputPage.js]: Some people may disagree with BR being used for this instead of UK

Comment (georgegillams) [reply docs/src/pages/PhoneInputPage/PhoneInputPage.js]: God I need more coffee
coffee

**Pull request merged**